src/stcal/jump/twopoint_difference.py

```diff
@@ -152,186 +157,203 @@ def find_crs(
         for grp in range(dat.shape[1]):
             if np.all(np.bitwise_and(gdq[integ, grp, :, :], dnu_flag)):
                 num_flagged_grps += 1
-    total_groups = nints if only_use_ints and nints else nints * ngrps - num_flagged_grps
-    if (ngrps < minimum_groups and only_use_ints and nints < minimum_sigclip_groups) or (
-        not only_use_ints and nints * ngrps < minimum_sigclip_groups and ngrps < minimum_groups
-    ):
+    if only_use_ints:
+        total_sigclip_groups = nints
+    else:
+        total_sigclip_groups = nints * (ngrps - num_flagged_grps)
+    total_groups = nints * (ngrps - num_flagged_grps)
+    total_diffs = nints * (ngrps - 1 - num_flagged_grps)
+    total_usable_diffs = total_diffs - num_flagged_grps
+    if ((ngrps < minimum_groups and only_use_ints and nints < minimum_sigclip_groups) or
+            (not only_use_ints and nints * ngrps < minimum_sigclip_groups and
+             total_groups < minimum_groups)):
         log.info("Jump Step was skipped because exposure has less than the minimum number of usable groups")
-        log.info("Data shape %s", dat.shape)
-        dummy = np.zeros((dataa.shape[1] - 1, dataa.shape[2], dataa.shape[3]), dtype=np.float32)
-
+        log.info("Data shape {}".format(str(dat.shape)))
```

Comment:
I'm not sure we really want low-level info like this coming out at the ``info`` level of logging. Seems more appropriate to have as ``debug`` level.

Reply:
I removed it.